### ISSUE TITLE
Gene information not included in toxcast 3.3

### ISSUE THREAD

**Issue by lukeloken** [USGS-R/toxEval#388]
https://github.com/USGS-R/toxEval/blob/6f3cf4feeec51b751582235c3c0943c148c7862e/R/toxEval.R#L124

Hey @ldecicco-USGS , I am looking at the `end_point_info` table associated with toxcast 3.3 I know you are working on updating this table and found a couple of issues with other columns. I noticed that this table does not have many gene's named or IDed. The table is also missing a few columns (e.g., intended_target_gene_symbol) that were in the version 3.2. 

Anyway, I thought I would make a note here and let me know if I can help resolve the issue. The file referenced in this post looks to be on your machine rather than being in the package files.

**Comment by ldecicco-USGS** [comment]
Yeah, you'll have to download the summary files locally. The data is here:
https://www.epa.gov/chemical-research/exploring-toxcast-data-downloadable-data
Then you download and unzip the "ToxCast & Tox21 Summary Files for invitroDBv3.3"

Similarly, if you wanted to get the ACC stuff, you'd need to download and unzip the "ToxCast & Tox21 Data Spreadsheet for invitroDBv3.3".
